hack/tests/e2e-helm-new.sh

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env bash
```

Comment:
IIRC we want to add all new e2e tests as ginkgo/gomega suites.

Reply:
Hi @estroz, 

The first version was with the tests done in go like you wish 👍 . However, @joelanford asked for we keep here the same tests in the shell just for we ensure now that all is 100% then we can do a follow with the go tests as well.